source/contracts/libraries/token/BasicToken.sol

```diff
@@ -28,6 +28,19 @@ contract BasicToken is ERC20Basic {
     }
 
     /**
+    * @dev allows internal token transfers
+    * @param _from The source address
+    * @param _to The destination address 
+    */
+    function internalTransfer(address _from, address _to, uint256 _value) internal returns (bool) {
+        balances[_from] = balances[_from].sub(_value);
+        balances[_to] = balances[_to].add(_value);
+        Transfer(_from, _to, _value);
+        emitTransferLogs(_from, _to, _value);
```

Comment:
At the moment, `emitTransferLogs(...)` in most cases emits `Transfer` event (as well as some other event).  Either we should change this function to only call `emitTransferLogs` and not also emit the `Transfer` event, or we need to tackle one or more of the other tasks in the ClubHouse issue and fix the other call sites.

If you prefer small isolated PRs, then I think removing the `Transfer` event from this function is the right course of action (and in hindsight, perhaps what you meant when you said "fired twice") and then in a follow-up PR address the "`Transfer` can be missed" problem.

Reply:
removed the Transfer call in internalTransfer. Currently working on adding unit tests.